packages/shared/src/lib/interest/intrestStore.ts

```diff
@@ -0,0 +1,68 @@
+import { shallowCompare } from '$lib/shallowCompare';
+
+export interface Interest {
+	_subscribe: () => () => void;
+}
+
+interface Subscription<Arguments> {
```

Comment:
Could we use `T` here instead of `Arguments` since it's standard practice?

Reply:
Is it a good standard practice?

Reply:
Taking a look at some of the type definitions in redux toolkit, they seem to use named type parameters in the vast majority of cases. The typescript documentation doesn't give a recommendation, but does avoid using single letter type parameter names in their examples. https://www.typescriptlang.org/docs/handbook/2/functions.html#guidelines-for-writing-good-generic-functions